doc/examples/jupyter_notebook_prototyping/src/moveit_notebook_tutorial.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4df8633e",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "Welcome to this tutorial on using jupyter notebooks with Moveit 2. A great benefit of being able to interact with MoveIt via a Python notebook is the ability to rapidly prototype code. We hope you find this interface intuitive and that you gain value from using MoveIt via Python notebooks.\n",
+    "\n",
+    "In this tutorial we will cover the following: \n",
+    "\n",
+    "* The required imports to run the notebook\n",
+    "* A motion planning example\n",
+    "* A teleoperation example\n",
+    "\n",
+    "If you have suggestions or feedback for this tutorial please post an issue on GitHub (https://github.com/ros-planning/moveit2_tutorials) and tag @peterdavidfagan."
```

Comment:
You sure you want to tell people to directly tag you in the issues? ;)

I personally wouldn't do this, haha.

Reply:
:) I am committed to fixing issues with the library so long as I can alongside my research projects. I am happy to have people ping me to make me aware of issues with the library.